tests/test_timeseriesflattener/test_add_values.py

```diff
@@ -486,3 +488,40 @@ def test_add_temporal_incident_binary_outcome():
         "outc_dichotomous_t2d_within_2_days_max_fallback_0",
     ]:
         pd.testing.assert_series_equal(outcome_df[col], expected_df[col])
+
+
+def test_add_text_data():
+    prediction_times_str = """dw_ek_borger,timestamp,
+                            746430.0,1970-05-01 00:00:00
+                            765709.0,1971-05-14 22:04:00
+                            """
```

Comment:
`dw_ek_borger` as floats? Resolves to str otherwise?

Reply:
Chosen to match two random ids in the synthetic data (which are floats for some reason)